### ISSUE TITLE
referring to the usecases file in the README file...

### ISSUE THREAD

**Issue by atrent** [shreyanshk/boottorrent#10]
I'm not sure about the usefulness of the usecases file for *external* users, I see it as an internal document for the development team, maybe here a brief summary of the hypothetical contexts can be described, what do you think?

**Comment by atrent** [comment]
a casual user needs to understand **what, where, when**, e.g.:
- what do I need to install if my setup is ...?
- where do I copy the boot files if I have a router?
- when (in which order) do I run all the pieces of software?

**Comment by atrent** [comment]
(see my comment in the README: the usecases file is not even referred in any other file!)

**Comment by shreyanshk** [comment]
Yes! I've added a comment as well: "I'll write usecases in simpler terms for a new user. They were linked before but were too technical, like you previously said. I'll rewrite and put them back as soon as I get an opportunity."

**Comment by atrent** [comment]
you may leave the usecases file as technical as it is, but a summarized description should be written down

**Comment by shreyanshk** [comment]
> what do I need to install if my setup is ...?
> where do I copy the boot files if I have a router?
I've tried to answer it in the frequently asked questions section. https://boottorrent.readthedocs.io/en/latest/faq.html

> when (in which order) do I run all the pieces of software?
I've tried to answer it in the quick start guide.

I've added usecases on the index page: https://boottorrent.readthedocs.io/en/latest/index.html

Please consider reading them and let me know what you think.
Thank you.

**Comment by atrent** [comment]
(commented the architecture file)

**Comment by shreyanshk** [comment]
I've simplified the use cases, the questions are answered in the FAQ section and added use case summary in the ./README.rst
Is that better?
